docs/operations/updating_the_stack.rst

```diff
@@ -1,133 +1,200 @@
 .. _operations-updating-the-stack:
 
```

Comment:
Just FYI, I there are some doc changes in [this recent PR](https://github.com/openstax/output-producer-service/pull/106/files) which may need to get massaged in with your updates to this page.